jobs/elasticsearch_alerts/templates/cluster.alerts

```diff
@@ -0,0 +1,37 @@
+ALERT ElasticUp
+  IF elasticsearch_up != 1
+  FOR <%= p('elasticsearch_alerts.cluster_down.evaluation_time') %>
+  LABELS {
+    severity = "alert",
+    value = "{{$value}}",
+  }
+  ANNOTATIONS {
+    summary = "Elasticsearch instance {{$labels.instance}} is not responding",
+    description = "Elasticsearch instance {{$labels.instance}} is not responding",
+  }
+
+ALERT ElasticClusterRed
+  IF elasticsearch_cluster_health_status = 2
+  FOR <%= p('elasticsearch_alerts.cluster_red.evaluation_time') %>
+  LABELS {
```

Comment:
Same as above. Please add the `service` label, update the `severity` to `warning` or `critical` and remove `value`.